core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -1537,6 +1542,26 @@ class EMConfig {
     @Cfg("Specify a path to save all executed sql commands to a file (default is 'sql.txt')")
     var saveExecutedSQLToFile : String = "sql.txt"
 
+    @Experimental
+    @Cfg("Whether to enable extra targets for responses, e.g., regarding nullable response, having extra targets for whether it is null")
+    var enableRPCExtraResponseTargets = false
+
+    @Experimental
+    @Cfg("Whether to enable customized responses indicating business logic")
+    var enableRPCCustomizedResponseTargets = false
+
+    @Experimental
+    @Cfg("Whether to generate RPC tests which is independent from EM driver.")
+    var enablePureRPCTestGeneration = false
+
+    @Experimental
+    @Cfg("Whether to generate RPC Assertions based on instance")
+    var enableRPCAssertionWithInstance = false
+
+    @Experimental
+    @Cfg("Specify a maximum number of data in a collection to be asserted in generated tests." +
+            "Note that a zero or negative value means all data in the collection should be asserted.")
+    var maxAssertionForDataInCollection = -1
```

Comment:
shouldn't `0` rather mean that no data is going to be asserted?

Reply:
it could be, then it means that only `size` of collection would be asserted.